policies/templates/attestations/vuln/vuln-maximum-scan-age-rego.yaml

```diff
@@ -24,7 +24,7 @@ spec:
           subjectRegExp: .*
       attestations:
         - name: vuln-report-scan-max-age
-          predicateType: https://cosign.sigstore.dev/attestation/vuln/v1
+          predicateType: cosign.sigstore.dev/attestation/vuln/v1
```

Comment:
this won't work with cosign v2

Reply:
I'm not sure how to resolve that, and it's unrelated to my change, so I just reverted it.

Reply:
We will need to handle both cosign versions. Let me share you an example.